tests/Functional/PluginManagerCommandsTest.php

```diff
@@ -107,7 +107,7 @@ public function testPluginsCommands()
         $this->assertCommandExists('self:plugin:migrate');
         $this->installTerminus2Plugins($testPluginPackages);
         $this->terminusWithStderrRedirected('self:plugin:migrate');
-        $pluginList = $this->terminusWithStderrRedirected('self:plugin:list');
+        $pluginList = $this->terminusWithStderrRedirected('self:plugin:list --format=json');
```

Comment:
@uberhacker This fixes failing "plugin" functional tests group - https://github.com/pantheon-systems/terminus/runs/4785452801?check_suite_focus=true